webpack.config.js

```diff
@@ -4,7 +4,8 @@
  * External dependencies
  */
 var webpack = require( 'webpack' ),
-	path = require( 'path' );
+	path = require( 'path' ),
+	config = require( 'config' );
```

Comment:
`config` is already declared 4 lines down from here as `require( './server/config' );`. We should eliminate one.